tests/bug-7241-02-pre8/test.yaml

```diff
@@ -0,0 +1,20 @@
+requires:
+  min-version: 7
+
+pcap: ../tls/tls-random/input.pcap
+
+args:
+- -k none
+
+checks:
+- filter:
+    count: 0
+    match:
+      alert.signature_id: 1
+      event_type: alert
+checks:
```

Comment:
This test passes because this is wrong syntax for `test.yaml`..
Here, we're checking for an event that matches a checks type dictionary in the yaml alongwith the attributes mentioned above. Since it does not exist, the count for such an event is 0..

Found when I was building upon this test and saw inconsistencies in non negation behavior.
1. Non negation behavior is just fine. My analysis was incorrect because I was building upon an incorrect test.
2. Negation behavior is buggy as shown by the ticket. This test does and should fail ideally on `main-7.0.x`.

Sending a PR fixing this..

Reply:
❤️ yaml